server/chat-plugins/mafia.ts

```diff
@@ -2569,6 +2635,46 @@ export const commands: ChatCommands = {
 			`/mafia revealas [player], [role] - Fakereveals the role of a player as a certain role. Requires host % @ # &`,
 		],
 
+		unidle: 'idle',
+		unaction: 'idle',
+		noresponse: 'idle',
+		action: 'idle',
+		idle(target, room, user, connection, cmd) {
```

Comment:
it might be worthwhile to show the player an error if they try to give this command arguments, in case they think that `/mafia action do whatever` will submit that to the host?